lib/chef/data_collector/run_end_message.rb

```diff
@@ -61,7 +62,7 @@ def construct_message(data_collector, status)
             "policy_group" => node&.policy_group,
             "start_time" => run_status.start_time.utc.iso8601,
             "end_time" => run_status.end_time.utc.iso8601,
-            "source" => solo_run? ? "chef_solo" : "chef_client",
+            "source" => solo_run? ? Chef::Dist::SOLOEXEC : CHEF::DIST::CLIENT,
```

Comment:
any serious implications? Could these be considered as "internals"?

Reply:
Leave this one the way it was